src/api/entityResolution/types.ts

```diff
@@ -395,3 +395,144 @@ export interface EntityResolutionMetrics {
    */
   possibleRelationships: number;
 }
+
+export interface ExplainWhyRecordParams extends IDataSourceParams {
+  /**
+   * The ID of the record to explain. It corresponds to the ID of the record node in the graph
+   * database.
+   */
+  recordId: string;
+}
+
+export interface ExplainWhyEntitiesParams extends IDataSourceParams {
+  /**
+   * The ID of the first entity to explain. It corresponds to the `entityId` property of the entity
+   * node in the graph database.
+   */
+  entityId: number;
+  /**
+   * The ID of the other related entity.
+   */
+  otherEntityId: number;
```

Comment:
what is entityId?  is it the node id? if so it should be a string and not number similar to recordId

Reply:
entityId is an id managed by senzing, you have it from the `ResolvedPerson/OrganisationEntity` node properties. It's a number because Senzing decided so.

Records are objects we send to senzing, so their id is what we decide, in our case it's the graph nodes and they are string because we decided so for LKE.